cli.js

```diff
@@ -37,6 +38,10 @@ if (program.formatter) {
   configuration.formatter = program.formatter || "json.js";
 }
 
+if (program.excluded && typeof(program.excluded) === "string") {
+	configuration.excluded = program.excluded.split(",");
```

Comment:
I would recommend to add a default value, e. g. empty array to avoid additional checks in other places

Reply:
I added a default value in the initialization of configuration